bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/contentfragment/ContentFragmentListImpl.java

```diff
@@ -91,6 +94,10 @@
     @ValueMapValue(name = ContentFragmentList.PN_PARENT_PATH, injectionStrategy = InjectionStrategy.OPTIONAL)
     private String parentPath;
 
+    @ValueMapValue(name = ContentFragmentList.PN_RESULT_LIMIT, injectionStrategy = InjectionStrategy.OPTIONAL)
+    @Default(intValues = DEFAULT_RESULT_LIMIT)
+    private int resultLimit;
```

Comment:
I would call the property `maxItems` to be consistent with the Core List component.